doc/devel/testing.rst

```diff
@@ -139,6 +139,23 @@ execution (such as created figures or modified rc params). The pytest fixture
 :func:`~matplotlib.testing.conftest.mpl_test_settings` will automatically clean
 these up; there is no need to do anything further.
 
+Random data in tests
+--------------------
+
+Random data can is a very convenient way to generate data for examples,
+however the randomness is problematic for testing (as the tests
+must be deterministic!).  To work around this set the seed in the test.
```

Comment:
"in the test*s* ?": typo or correct English.

Reply:
tweaked a bit.  The seed needs to be set in each test, just just once.